app/views/registration/ConfirmUpdateNextTaxYear.scala.html

```diff
@@ -41,7 +42,7 @@ <h1 id="title" class="govuk-heading-xl">
     </h1>
 
     <div class="data">
-        @formWithCSRF(action = controllers.registration.routes.ManageRegistrationController.addNextYearRegisteredBenefitTypes, Symbol("onsubmit") -> "gaEventConfirmAddBenefits()") {
+        @formWithCSRF(action = controllers.registration.routes.ManageRegistrationController.addNextYearRegisteredBenefitTypes) {
```

Comment:
I think I need this explained why it is removed :)

Reply:
the scripts to add the google analytics events were removed as part of a ticket 2yrs ago (May/2021) after that bit of clean-up was done however the cleanup was not done properly. And this is the remnants of that clean-up. We should have been hitting a console error in production as well cause of this, cause we're trying to call a function that's not available any more